terraform/staging/ecr.tf

```diff
@@ -0,0 +1,5 @@
+# Currently used by auth service for pulling AWS telemetry sidecar
+resource "aws_ecr_pull_through_cache_rule" "ecr_public2" {
```

Comment:
Why'd you name it "ecr_public2" when it's in a separate account from the other one?

Reply:
Just a mistake, fixed :+1: